src/pods/toolbar/components/add-collection/set-off-set-zoom-to-coords.hook.tsx

```diff
@@ -0,0 +1,23 @@
+import { useCanvasViewSettingsContext } from '@/core/providers/canvas-view-settings';
```

Comment:
The PR contains merge conflicts could you fix it? Thanks

Reply:
Fixed conflicts, ready to merge!